### PR TITLE
Refactor elbow plot into separate container and rename view

### PR DESCRIPTION
**Pull request by cosa65**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-522?atlOrigin=eyJpIjoiN2Y1NTRkZjBiZWJiNDgxMWEzOTEyMGY2OGFkNjMzNmUiLCJwIjoiaiJ9

#### Link to staging deployment URL 
https://ui-imm3irch1y5e57hfsi3ncsdrge.scp-staging.biomage.net/

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [x] Tested locally with Inframock
- [x] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration/DataIntegration.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import React, { useState } from 'react';
+import {
+  Row, Col, Space, Select,
+  InputNumber, Form, Checkbox, Button, Tooltip,
+} from 'antd';
+import _ from 'lodash';
+import {
+  InfoCircleOutlined,
+} from '@ant-design/icons';
+// import _ from 'lodash';
```
Comment (ivababukova) [inline src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration/DataIntegration.jsx]: no need for this commented out import

File: src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration/DataIntegration.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          </Form.Item>
+          <Form.Item label='Exclude genes categories:'>
+            <Checkbox.Group onChange={() => { }}>
+              <Space direction='vertical'>
+                <Checkbox value='ribosomal'>ribosomal</Checkbox>
+                <Checkbox value='mitochondrial'>mitochondrial</Checkbox>
+                <Checkbox value='cellCycle'>cell cycle</Checkbox>
+              </Space>
+            </Checkbox.Group>
+          </Form.Item>
```
Comment (ivababukova) [inline src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration/DataIntegration.jsx]: I would also create a new component for this, similarly to how we have `CalculationConfig` for the Configure Embedding, and I would put these there. In that way, during subsequent work we can have multiple people working on the ui for data integration - someone working on plot styling and someone working on this config for example.

File: src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration/plots/ElbowPlot.jsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+          },
+        },
+      },
+    ],
+    title:
+    {
+      text: { value: plotConfig.titleText },
+      anchor: { value: plotConfig.titleAnchor },
+      font: { value: plotConfig.masterFont },
+      dx: 10,
```
Comment (ivababukova) [inline src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration/plots/ElbowPlot.jsx]: what is this dx and why its value is not in the config?

Comment (cosa65) [reply src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration/plots/ElbowPlot.jsx]: It is the horizontal offset of the title, I'll add it to the config to make it more consistent.

**Review by ivababukova — APPROVED** [review]

**Pull request merged**